src/components/artwork/details.tsx

```diff
@@ -9,22 +9,17 @@ export interface DetailsProps extends React.HTMLProps<ArtworkDetails> {
 }
 
 export class ArtworkDetails extends React.Component<DetailsProps, null> {
-
   artistLine() {
-    const cultural_maker = this.props.artwork.cultural_maker
-    const artists = this.props.artwork.artists
+    const { cultural_maker, artists } = this.props.artwork
 
     if (cultural_maker) {
-      return (<div><strong>{ cultural_maker }</strong></div>)
+      return <div><strong>{ cultural_maker }</strong></div>
     } else if (artists && artists.length) {
-      let artistsEl = []
-      for (let i = 0; i < artists.length; i++) {
-        artistsEl.push(<TextLink href={ artists[i].href } key={ i }>{ artists[i].name }</TextLink>)
-        if (i !== artists.length - 1) {
-          artistsEl.push(<span>,</span>)
-        }
-      }
-      return (<div><strong>{ artistsEl }</strong></div>)
+      let artistLine = artists
+        .map(artist => <TextLink href={ artist.href } key={ artist.__id }>{ artist.name }</TextLink>)
```

Comment:
Interesting. It might be just `undefined` but then React should complain about duplicate keys. Not sure why, but I'll add `__id` to the fragment.